### ISSUE TITLE
Arm support via linuxserver/plex

### ISSUE THREAD

**Issue by benmichael** [billimek/billimek-charts#320]
As this project largely serves home projects, a fair number of the audience may be using arm based SBC clusters (especially with there now being an 8GB Raspberry Pi).

I am aware that the Plex maintained docker images only cater to x86, however the linuxserver.io images include arm images. 

While it may require significant effort to cleanly migrate, my proposal would be to do so, so as to cater to a wider audience. 

While it would mean moving away from the official images, linuxserver.io has a good reputation for maintaining high quality images. 

https://hub.docker.com/r/linuxserver/plex/

**Comment by billimek** [comment]
Good call-out @benmichael!  I looked at the linuxserver/plex container image configuration and believe that it may be 'compatible' with the settings already in-place (volume mount locations, env variables) from the 'official' image.

It may be something easy to switch as a helm setting choice by setting something like this:

```yaml
image:
  repository: linuxserver/plex
  tag: 1.19.5.3112-b23ab3896-ls112
```

It may be worth trying with a local install of the plex chart to see if it behaves properly with the linuxserver image?

**Comment by benmichael** [comment]
Excellent, I'll give that a try early next week (it would be earlier, but life). I'll let you know what I find.

**Comment by benmichael** [comment]
Finally got around to testing this, and everything seems to be working with `linuxserver/plex`. I'll submit a PR
